evaluation.py

```
@@ -71,16 +104,18 @@ def main():
         uid, test_idx = test_input_to_uid_test_idx[grid_2d_to_tuple(test_input)]
         top2 = []
         for response in responses[0:2]:
-            response = response.strip("`")
+            response = extract_code_blocks(response)
+            # response = response.strip("`")
             grid = None
             try:
                 grid = color_grid_to_int_grid(response)
             except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)